src/styles/components/home/_home.scss

```diff
@@ -421,7 +421,7 @@
       position: fixed;
     }
 
-    > ul {
+    > .tokenList {
```

Comment:
You verified this won't conflict somewhere else? Mainly concerned as this is indented CSS styling

Reply:
So far as token list isn't used anywhere else, it won't conflict